fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalAggregate.java

```diff
@@ -390,4 +390,9 @@ public ImmutableSet<FdItem> computeFdItems() {
 
         return builder.build();
     }
+
+    @Override
+    public void computeEqualSet(FunctionalDependencies.Builder fdBuilder) {
+        fdBuilder.addEqualSet(child().getLogicalProperties().getFunctionalDependencies());
```

Comment:
agg output not equal with its child, so just add is ok?

Reply:
It's ok. we don't need to prune slot for equal relation